src/main/java/programmers/org/voucher/exception/ErrorMessage.java

```diff
@@ -0,0 +1,16 @@
+package programmers.org.voucher.exception;
+
+public enum ErrorMessage {
+    COMMAND_ERROR_MESSAGE("유효하지 않은 커맨드입니다."),
+    VOUCHER_ERROR_MESSAGE("유효하지 않은 바우처 타입입니다.");
+
+    private final String errorMessage;
```

Comment:
네이밍을 간단하게 message 는 어떠한가요?
```suggestion
    private final String message;
```

Reply:
1ac1e90 커밋 피드백 반영
저도 message가 더 간결하고 좋은 것 같습니다! 수정하였습니다😉